Consul/Agent.cs

```diff
@@ -386,9 +387,19 @@ public async Task<string> GetNodeName(CancellationToken ct = default(Cancellatio
         /// Services returns the locally registered services
         /// </summary>
         /// <returns>A map of the registered services and service data</returns>
-        public Task<QueryResult<Dictionary<string, AgentService>>> Services(CancellationToken ct = default(CancellationToken))
+        public async Task<QueryResult<Dictionary<string, AgentService>>> Services(CancellationToken ct = default(CancellationToken))
         {
-            return _client.Get<Dictionary<string, AgentService>>("/v1/agent/services").Execute(ct);
+            return await Services(null, ct);
```

Comment:
It is better to use `await` even for simple pass-through invocations like this one. When the call is not awaited then it will be not visible on the call stack (if there is an exception being thrown within that call).